packages/flutter/lib/src/rendering/shifted_box.dart

```diff
@@ -515,6 +515,23 @@ class RenderPositionedBox extends RenderAligningShiftedBox {
   }
 }
 
+/// How much space should be occupied by the [OverflowBox].
+///
+/// See also:
+///
+///  * [OverflowBox], the widget using this enum.
+enum OverflowBoxFit {
+  /// The widget will size itself to be as large as the parent allows.
+  max,
+
+  /// The widget will follow the child's size.
+  ///
+  /// More specifically, the render object will size itself to match the size of
+  /// its child within the constraints of its parent or be as small as the
+  /// parent allows if no child is set.
+  passthrough,
```

Comment:
This name isn't great. Other suggestions: `min` (as in be as small as possible without clipping the child), or `deferToChild` to be more explicit.